containers/base/build.sh

```diff
@@ -20,16 +20,18 @@
 
 # Build the docker image
 if [ -n "$1" ]; then
-  src_pydatalab="$1"
+  src_pydatalab=$(realpath "$1")
 
   # Append trailing / if needed.
   if [ "${src_pydatalab: -1}" != "/" ]; then
```

Comment:
The trailing slash is required by the rsync command, which behaves differently if it is not there. I moved adding the trailing slash to the rsync line.